department-of-veterans-affairs/caseflow
# Added a space between the intake and create record buttons

Resolves [There should be a space between the Intake appeal and the Create record button on the Review Package page.](https://jira.devops.va.gov/browse/APPEALS-50876)

# Description
Added space between Intake appeal and the Create record button

## Acceptance Criteria

## Testing Plan
1. Log into Caseflow as [INBOUND_OPS_TEAM_ADMIN_USER (VACO]
2. Select Queue and then Correspondence Cases
3. Select any Veteran under the Veteran Details Column with NOD type
4. On the Review Package page, scroll down to the bottom of the page.
5. Observe there should be a space between the _Intake appeal_ and the _Create record_ button on the Review Package page.




# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|
<img width="358" alt="Screenshot 2024-07-31 at 10 33 42 AM" src="https://github.com/user-attachments/assets/22b71527-2e1c-47f0-b1e8-0c03a5259826">
---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
